.github/workflows/build-sphinx.yml

```diff
@@ -32,6 +32,30 @@ jobs:
             - name: Install python dependencies
               run: python -m pip install -r requirements.txt
 
+            - name: Check for unused files

```

Comment:
I tried thinking of a situation where we wouldn't mind having an unused file hanging around and I couldn't really come up with anything. I think it's fine to be strict and clean about this until a situation arises that makes us think differently.

Reply:
Agree, they were really piling up and I was using this script to find files to delete anyway, so I figured I'd add it to the workflow